st2client/st2client/models/service_registry.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from st2client.models import core
+
+__all__ = [
+    'ServiceRegistry',
+
+    'ServiceRegistryGroup',
+    'ServiceRegistryMember'
+]
+
+
+class ServiceRegistry(core.Resource):
+    _alias = 'service-registry'
+    _display_name = 'service registry'
```

Comment:
Please capitalize first letters.

Reply:
This attribute is really just a place holder, it's not used anywhere.

The class (model) is just needed to get ``st2 service-registry foo bar`` command structure to work since our, not so ideal CLI code, depends on it.

I can remove unused attributes all together.

Reply:
Ok these attributes are still here.  So, placeholder or not, please capitalize `service registry` to `Service Registry`. This is so the values are consistent with other models.